### PR TITLE
Add rsid to SV hist output

### PR DESCRIPTION
**Pull request by mike-w-wilson**
The SV team repurposes the rsid field to be a unique identifier. This adds the field into the output for use by the browser team.

### CODE REVIEW

**Comment by mike-w-wilson** [comment]
Screenshot 2023-05-03 at 8 44 10 AM
This is a unique fields

File: gnomad_qc/v3/annotations/generate_sv_hists.py [hunk 1]
```diff
@@ -137,6 +138,7 @@ def get_sex_and_autosome_mt() -> hl.MatrixTable:
         reference_genome="GRCh38",
         min_partitions=300,
         force_bgz=True,
+        array_elements_required=False,
```
Comment (ch-kr) [inline gnomad_qc/v3/annotations/generate_sv_hists.py:141]: why is this required now (and not previously)?

**Review by ch-kr — APPROVED** [review]
one small question about the `array_elements_required` param, otherwise LGTM!

File: gnomad_qc/v3/annotations/generate_sv_hists.py (hunk 1 above)
Comment (mike-w-wilson) [reply gnomad_qc/v3/annotations/generate_sv_hists.py:141]: Sorry -- this was a runtime error, didnt come up in my small test dataset but there was a missing value in the larger full dataset, and I fixed it locally for the original run.
